src/org/exist/client/InteractiveClient.java

```diff
@@ -188,6 +188,9 @@
     protected XmldbURI path = XmldbURI.ROOT_COLLECTION_URI;
     private Optional<Writer> lazyTraceWriter = Optional.empty();
 
+    private static final NamedThreadGroupFactory clientThreadGroupFactory = new NamedThreadGroupFactory("javaadminclient");
```

Comment:
better name = java-adminclient ?

Reply:
just a suggestion

Reply:
Okay fixed. I changed it to "java-admin-client"